src/main/java/org/opensearch/knn/index/mapper/KNNVectorFieldMapperUtil.java

```diff
@@ -74,6 +78,43 @@ public static void validateByteVectorValue(float value) {
         }
     }
 
+    /**
+     * Validate the float vector value and throw exception if it is not a number or not in the finite range
+     * or is not within the FP16 range of [-65504 to 65504].
+     *
+     * @param value float vector value
+     */
+    public static void validateFP16VectorValue(float value) {
```

Comment:
Can we add unit tests for these 2 methods?

Reply:
updated